packages/block-library/src/quote/v2/transforms.js

```diff
@@ -29,7 +29,7 @@ const transforms = {
 						query: {
 							content: {
 								type: 'string',
-								source: 'text',
```

Comment:
I've commited this here because it's the same fix I did for the migration. Thought it doesn't deserve a separate PR.